src/utils/OpenPepXLLF.cpp

```diff
@@ -413,71 +439,61 @@ class TOPPOpenPepXLLF :
 
     vector<PeptideIdentification> peptide_ids;
```

Comment:
your main method is almost 1000 LOC, this is clearly too long. please consider extracting some functions

Reply:
@enetz I agree with Hannes. I think a good option would be to move most of the code in main to an algorithm class. This would already get rid of some boilerplate parameter code / reading and writing of input output files. From my side, this could also be done after a merge as checks are currently passing and these changes would further expand this already big PR.

Reply:
There is also still some duplicated functionality between the two binaries. I will start by abstracting some more static functions that the tools can share into the OPXLHelper class. Then move the main functions in algorithm classes.
And I also agree that this should be a separate PR.

Reply:
Sounds good. Just consider putting that code into dedicated classes and not everything into a Helper class. The reason is, that Helper classes are considered an anti-pattern under most circumstances as they don't define a clear class responsibility (rather a dump for methods ;) ).

Reply:
In this case I would rather have a dump for methods, since this Helper class is basically just abstraction for reused functionality between two main classes. Howerver I will think about what could be usable outside of those main functions and put them in other classes.